manifests/init.pp

```diff
@@ -121,7 +121,11 @@
 #
 class simp (
   Hash                            $scenario_map,
-  Simp::Scenario                  $scenario                   = 'simp',
+  Enum['simp',
```

Comment:
@jeefberkey Not worried about ugly, worried about accurate. `puppet strings` will pick this up properly and present it to the user. We can do way too much work to make it pretty and document it two places later.